TERMINOLOGY.md

```diff
@@ -11,17 +11,17 @@ The fairlearn package largely follows the [terminology established by scikit-lea
 
 ## Fairness of AI systems
 
-AI systems can behave unfairly for a variety of reasons. Sometimes it is because of societal biases reflected in the training data and in the decisions made during the development and deployment of these systems. In other cases, AI systems behave unfairly not because of societal biases, but because of characteristics of the data (e.g., too few data points about some group of people) or characteristics of the systems themselves. Because it can be hard to distinguish between these reasons (and these reasons are not mutually exclusive and often exacerbate one another), we define whether an AI system is behaving unfairly in terms of its impact on people – i.e., in terms of harms – and not in terms of specific causes, such as societal biases, or in terms of intent, such as prejudice.
+AI systems can behave unfairly for a variety of reasons. Sometimes societal biases are reflected in the training data and in the decisions made during the development and deployment of these systems. In other cases, AI systems behave unfairly because of characteristics of the data (e.g. too few data points about some group of people) or characteristics of the systems themselves. It can be hard to distinguish between these reasons, especially since they are not mutually exclusive and often exacerbate one another. Therefore we define whether an AI system is behaving unfairly in terms of its impact on people – i.e. in terms of harms – and not in terms of specific causes, such as societal biases, or in terms of intent, such as prejudice.
 
```

Comment:
Let's revert the edits at the beginning of the paragraph, because they subtly change the meaning away from the intended--the goal was to highlight that unfairness may arise due to societal biases or in their absence--that's lost right now.

I like the rewrite that starts with "It can be hard" but a few issues there need correcting. Please use the following:

It can be hard to distinguish between these reasons, especially since they are not mutually exclusive and often exacerbate one another. Therefore, we define whether an AI system is behaving unfairly in terms of its impact on people&mdash;i.e., in terms of harms&mdash;and not in terms of specific causes, such as societal biases, or in terms of intent, such as prejudice.

As a convention, we should write "i.e.," and "e.g.," with the comma:
https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Abbreviations/faq0047.html

Reply:
Interesting, this is in direct conflict with Richard's feedback :-D

Reply:
I won't argue with the style manual.